storage/mutable_segment.go

```diff
@@ -574,7 +573,7 @@ func (s *mutableSeg) writeTimestampFieldWithLock(docID int32, val int64) {
 func (s *mutableSeg) writeRawDocSourceFieldWithLock(docID int32, val []byte) {
 	v := field.ValueUnion{
 		Type:      field.StringType,
-		StringVal: safe.ToString(val),
+		StringVal: val,
```

Comment:
do we want to switch `StringVal` to type []byte as well?

Reply:
It's already converted to `[]byte` in the struct def

Reply:
sorry I meant to change the field to `bytesVal`